test/test_suite.sh

```diff
@@ -14,13 +14,34 @@ echo "#########################################"
 
 docker --version
 
-docker load -i "../artifacts/$WIKIBASE_BUNDLE_IMAGE_NAME.docker.tar.gz"
+export SUITE=$1
+
+if [ -z "$SUITE" ]; then
+    echo "SUITE is not set"
+    exit 1
+fi
+
+if [ -z "$DATABASE_IMAGE_NAME" ]; then
+    export DATABASE_IMAGE_NAME="$DEFAULT_DATABASE_IMAGE_NAME"
+fi
+
+# select image based on prepended suite name
+if [[ $SUITE == base__* ]]; then
+    WIKIBASE_TEST_IMAGE_NAME="$WIKIBASE_IMAGE_NAME"
+else
+    WIKIBASE_TEST_IMAGE_NAME="$WIKIBASE_BUNDLE_IMAGE_NAME"
+
+    # load additional bundle images
+    docker load -i "../artifacts/$ELASTICSEARCH_IMAGE_NAME.docker.tar.gz"
+    docker load -i "../artifacts/$QUICKSTATEMENTS_IMAGE_NAME.docker.tar.gz"
+fi
+
+export WIKIBASE_TEST_IMAGE_NAME
+
+# load default images
+docker load -i "../artifacts/$WIKIBASE_TEST_IMAGE_NAME.docker.tar.gz"
 docker load -i "../artifacts/$QUERYSERVICE_IMAGE_NAME.docker.tar.gz"
 docker load -i "../artifacts/$QUERYSERVICE_UI_IMAGE_NAME.docker.tar.gz"
-docker load -i "../artifacts/$ELASTICSEARCH_IMAGE_NAME.docker.tar.gz"
```

Comment:
We could also avoid downloading this? but it's all baked into the build artefacts. Maybe not worth the extra complexity

Reply:
On one side this generates alot of traffic, but we are still downloading everything in ~30 secs. I'd say it's not worth the effort of splitting them out.